### ISSUE TITLE
Accessibility - Web - French - Header/Top Banner Read in English Instead of French

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#101]
All the header/top banner items (ILAB logo, application name, and DOL seal) are read as English words along with the search button and the information button on the bottom of the home page and other pages instead of the chosen language. Those should be translated into the language the person has chosen and not English. Some examples are ILAB (Bureau of International Labor Affairs) needs to be translated into French along with "United States Department of Labor" seal, search button, information button, etc.

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
JAWS seems to be reading the header/top banner items (ILAB logo, application name, and DOL seal) along with the search button and the information button on the bottom of the home page in French instead of English. So I'm assuming the translations are working that were added since I don't speak French.

I'm going to close this issue.
